docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -0,0 +1,254 @@
+
+To support live regions, the following APIs have been added to WPF:
+
+- The <xref:System.Windows.Automation.AutomationElementIdentifiers.LiveSettingProperty?displayProperty=nameWithType> and <xref:System.Windows.Automation.AutomationElementIdentifiers.LiveRegionChangedEvent?displayProperty=nameWithType> fields, which identify the **LiveSetting** property and the **LiveRegionChanged** event. They can be set by using XAML.
+
+- The **AutomationProperties.LiveSetting** attached property, which informs the screen reader of the importance of the UI change to the user.
+
+- The <xref:System.Windows.Automation.AutomationProperties.LiveSettingProperty?displayProperty=nameWithType> property, which identifies the **AutomationProperties.LiveSetting** attached property.
+ 
+- The <xref:System.Windows.Automation.Peers.AutomationPeer.GetLiveSettingCore%2A?displayProperty=nameWithType> method, which can be overridden to provide a **LiveSetting** value.
+
+- The <xref:System.Windows.Automation.AutomationProperties.GetLiveSetting%2A?displayProperty=nameWithType> and <xref:System.Windows.Automation.AutomationProperties.SetLiveSetting%2A?displayProperty=nameWithType> methods, which get and set a **LiveSetting** value.
+ 
+- The <xref:System.Windows.Automation.AutomationLiveSetting?displayProperty=nameWithType> enumeration, which defines the following possible **LiveSetting** values:
+
+   - <xref:System.Windows.Automation.AutomationLiveSetting.Off?displayProperty=nameWithType>. The element does not send notifications if the content of the live region has changed.   
+   - <xref:System.Windows.Automation.AutomationLiveSetting.Polite?displayProperty=nameWithType>. The element sends non-interruptive notifications if the content of the live region has changed.   
+
+  - <xref:System.Windows.Automation.AutomationLiveSetting.Assertive?displayProperty=nameWithType>. The element sends interruptive notifications if the content of the live region has changed.   
+
+You can create a LiveRegion by setting the **AutomationProperties.LiveSetting** property on the element of interest, as shown in the following example:   
+
+```xaml
+<TextBlock Name="myTextBlock" AutomationProperties.LiveSetting="Assertive">announcement</TextBlock>
+```
+
+When the data in the live region changes and you need to inform a screen reader, you explicitly raise an event, as shown in the following sample.
+
+```csharp
+var peer = FrameworkElementAutomationPeer.FromElement(myTextBlock);
+
+peer.RaiseAutomationEvent(AutomationEvents.LiveRegionChanged);
+```
+
+```vb
+Dim peer = FrameworkElementAutomationPeer.FromElement(myTextBlock)
+peer.RaiseAutomationEvent(AutomationEvents.LiveRegionChanged)
+
+```
+
+**High contrast**
+
+Starting with the .NET Framework 4.7.1, improvements in high conrast have been made to various WPF controls. They are now visible when the <xref:System.Windows.SystemParameters.HighContrast%2A> theme is set. These include:
```

Comment:
typo: conrast => contrast.

Reply:
Thanks, @BillWagner. I'll add it to the TOC.